src/components/Visualization.tsx

```diff
@@ -27,7 +29,11 @@ interface IVisualization {
 const Visualization = ({ toggleCatalog }: IVisualization) => {
   // `nodes` is an array of UI-specific objects that represent
   // the Integration.Steps model visually, while `edges` connect them
-  const defaultViewport: Viewport = { x: 10, y: 15, zoom: 1.2 };
+  const defaultViewport: Viewport = {
+    x: window.innerWidth / 2 - 80,
+    y: window.innerHeight / 2 - 160,
```

Comment:
why 80 and 160?

Reply:
That is the size of the node, which is fixed regardless of window size. And 160 is because it counts it from the center of the node, so you have to double it. I can change it to the use node width variable maybe so it's clearer.

Reply:
thanks for the answer.